jhipster-dependencies/pom.xml

```diff
@@ -81,6 +81,7 @@
         <testcontainers.version>1.11.3</testcontainers.version>
         <xmemcached.version>2.4.6</xmemcached.version>
         <xmemcached-provider.version>4.1.3</xmemcached-provider.version>
+        <redisson.version>3.11.1</redisson.version>
```

Comment:
@Shaolans Please move this property after <prometheus-simpleclient.version>, properties should be by default ordered by name

Reply:
Done thanks !